openads/tests/test_import.py

```diff
@@ -204,6 +205,38 @@ def test_import_constraints(self):
         self.assertEqual(results["COUNT_FEATURES"], 71)
         self.assertEqual(results["COUNT_NEW_CONSTRAINTS"], 0)
 
+        # INSEE should match the geojson
+        sql = "SELECT codeinsee FROM openads.geo_contraintes GROUP BY codeinsee;"
+        result = connection.executeSql(sql)
+        self.assertEqual(len(result), 1)
+        self.assertEqual(result[0][0], "80016")
+
+        # Empty geo_constraints
+        sql = "TRUNCATE openads.geo_contraintes RESTART IDENTITY;"
+        connection.executeSql(sql)
+
+        params["CHAMP_INSEE"] = "WRONGINSEE"
+        alg = "{}:data_constraints".format(provider.id())
+        results = processing.run(alg, params)
+
+        # 0 because the INSEE code is 80999
```

Comment:
Est-ce que le commentaire ne devrait-il pas être `# 0 because the INSEE code is WRONGINSEE` ?

Reply:
This INSEE code 80999 is in the `WRONGINSEE` column. I can improve.
I can update